pom.xml

```diff
@@ -220,6 +225,16 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.6</version>
```

Comment:
Have to be [careful here](https://jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/) since this is part of Core. Probably you meant to use `<exclude>s` instead.